modules/govuk_jenkins/templates/jobs/smokey.yaml.erb

```diff
@@ -30,9 +30,30 @@
           auth-token: <%= @environment_variables['SECOND_LINE_SLACK_AUTH_TOKEN']%>
           build-server-url: <%= @slack_build_server_url %>
           room: <%= @slack_room %>
-      - email:
+      - email-ext:
           recipients: govuk-ci-notifications@digital.cabinet-office.gov.uk
-          send-to-individuals: true
+          failure: true
+          presend-script: |
```

Comment:
Is there a more sensible way to have this groovy script in a separate file, and then include that file, rather than have it in the config itself?